test/mosaic-integration-tests/src/test/java/org/eclipse/mosaic/test/CommunicationSimulatorsIT.java

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright (c) 2020 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import org.eclipse.mosaic.rti.TIME;
+import org.eclipse.mosaic.starter.MosaicSimulation;
+import org.eclipse.mosaic.test.junit.LogAssert;
+import org.eclipse.mosaic.test.junit.MosaicSimulationRule;
+
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+/**
+ * This integration tests validates the core functionalities of the native MOSAIC communication simulators
+ * <b>Cell</b> and <b>SNS</b>.
+ * Functionalities tested:
+ * <ul>
+ *     <li/> Proper handling of different simulation entities in the simulators
+ *     <li/> Communication using Cell and Adhoc modules
+ *     <li/> Activation of Communication Modules
+ *     <li/> CAM building and sending
+ *     <li/> TCP acknowledgement test
+ * </ul>
+ * Units have the following roles:
+ * <ul>
+ *     <li/> veh0: Sending Cams
+ *     <li/> veh1: Receiving and returning RoundTripMessage
+ *     <li/> veh2: Sending Cams
+ * </ul>
+ */
+public class CommunicationSimulatorsIT {
```

Comment:
@kschrab Naming okay?

Reply:
I would prefer "SendAndReceiveIT" or even "CellSnsSendAndReceiveIT", as "CommunicationSimulatorsIT" is too general ( e.g. OMNeT++/ns-3 are not tested here).

Reply:
Well there is some communication via the SNS as well, so I would prefer `SendAndReceiveIT`. Should the scenario be called
``SendAndReceive` analogously?

Reply:
Yes